snmp/datadog_checks/snmp/data/default_profiles/hpe-msa.yaml

```diff
@@ -0,0 +1,155 @@
+extends:
+  - _base.yaml
+  - _generic-if.yaml
+  - _hp-base.yaml
```

Comment:
I'm wondering if it makes sense to import `_hp-base.yaml` since we are not sure all the metadata applies to this device: https://github.com/DataDog/integrations-core/blob/e79c81a813356a2fdd558e5c9395a7d52b290f6d/snmp/datadog_checks/snmp/data/default_profiles/_hp-base.yaml#L6-L25 (do we?)

Metadata from CPQSINFO-MIB